### PR TITLE
Website Release: New site content and languages

### PR DESCRIPTION
**Pull request by PardusEidolon**
## 29-04-22 - Changelog:
- More Indonesian content added
- Dependencies for website updated
- Intial fund 8 Catalyst TV episodes added to website
- New filipino language added to site with intial translations

### CODE REVIEW

**Review by andrecahyadi — COMMENTED** [review]
During the review, if I found any typos, I click the Viewed check box and then just write a comment?
Does the creator then need to revise & re-commit?

File: content/fil/stakepool/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Stake Pool"
+linkTitle: "Stakepool"
```
[the remark this answers is not in the document]
Comment (markibanez) [reply content/fil/stakepool/_index.md:3]: Not sure what the correct spelling is. I suppose it is stake pool but I use both interchangeably.

File: content/fil/docs/_index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Dokumentasyon"
```
[the remark this answers is not in the document]
Comment (markibanez) [reply content/fil/docs/_index.md:2]: Tagalog spelling of the Spanish word documentación

File: content/fil/stakepool/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrecahyadi) [reply content/fil/stakepool/_index.md:3]: Hi Mark, nice to know you :)
Yeah, I think most people write it, stake pool.

File: content/fil/docs/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrecahyadi) [reply content/fil/docs/_index.md:2]: I see, ok. 
Anyway, this is also my first time reviewing a pull request. Jack is the senior one, haha.

**Comment by PardusEidolon** [comment]
> During the review, if I found any typos, I click the Viewed check box and then just write a comment? Does the creator then need to revise & re-commit?

Youll have to patch the main branch with another pull request merging into main for it to up this PR

File: content/fil/docs/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (PardusEidolon) [reply content/fil/docs/_index.md:2]: > I see, ok. Anyway, this is also my first time reviewing a pull request. Jack is the senior one, haha.

I made you guys the reviewers so you have seniority over me to be honest 😂

File: content/fil/stakepool/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PardusEidolon) [reply content/fil/stakepool/_index.md:3]: my bad if I missed this in the inital PR for the filipino content, @markibanez oops

We can fix it up in the next patch, I'll throw up an issue as a reminder.

[the remark this answers is not in the document]
Comment (PardusEidolon) [reply content/fil/stakepool/_index.md:3]: This is fine for now

**Comment by PardusEidolon** [comment]
if Eveything checks out, im going to push this in to the release build @markibanez @andrecahyadi

like this if your okay or add a comment if you still want to make changes

**Pull request merged**